illumos-utils/src/zfs.rs

```diff
@@ -220,18 +220,11 @@ pub struct Zfs {}
 
 /// Describes a mountpoint for a ZFS filesystem.
 #[derive(Debug, Clone)]
-pub enum Mountpoint {
-    #[allow(dead_code)]
-    Legacy,
```

Comment:
It simplifies logic elsewhere if this is "always a path". I went ahead and updated this, since there appear to be no users of "legacy".